src/components/OnboardingMenu/OnboardingMenu.tsx

```diff
@@ -128,3 +131,9 @@ export const OnboardingMenu = ({
         </div>
     );
 };
+
+export const OnboardingMenu = Object.assign(OnboardingMenuParent, {
+    Item: OnboardingMenuItem,
+    ItemText: OnboardingMenuItemText,
+    ItemImage: OnboardingMenuItemImage,
+});
```

Comment:
We should also add `OnboardingMenu.displayName = 'OnboardingMenu';`, otherwise it would be `OnboardingMenuParent `